tests/integration/telemetry/util.go

```diff
@@ -46,39 +39,3 @@ func PromDumpWithAttributes(cluster resource.Cluster, prometheus prometheus.Inst
 
 	return ""
 }
-
-func SendTraffic(ingress ingress.Instance, t *testing.T, msg, url, extraHeader string, calls int64) *fhttp.HTTPRunnerResults {
```

Comment:
i didn't realize we had completely switched away from using this as a traffic generator. I wonder if this "utility" can go away, and be refactored under `tests/integration/telemetry/prometheus` or something as all that is left is the Dump methods.

Reply:
Was looking into making sure I wouldn't break istio's usage of fortio as libray in upcoming update but see it is gone... is there some background as to why/what is the replacement for exact/programmable traffic generation in tests nowadays in istio's code base?

Reply:
pkg/test/echo

Reply:
yeah I found https://github.com/istio/istio/tree/master/pkg/test/framework/components/echo interesting!
thx